### PR TITLE
doc: update libraries used

### PR DESCRIPTION
**Pull request by teolemon**
This is a first guess based on Github code search.
@VaiTon @naivekook could you possibly improve this PR with your knowledge of what we added ?

### CODE REVIEW

**Comment by naivekook** [comment]
we have duplicated lines 64 and 69

File: README.md [hunk 1]
```diff
@@ -105,21 +105,15 @@ If you spot any libraries we added or we don't use anymore, feel free to update
 
 - [Dagger 2](https://github.com/google/dagger) - A fast dependency injector for Android and Java
 - [Retrofit](https://square.github.io/retrofit/) - Retrofit turns your REST API into a Java interface
-- [OkHttp](https://github.com/square/okhttp) - An HTTP+SPDY client for Android and Java applications
 - [Mockito](https://github.com/mockito/mockito) - Most popular Mocking framework for unit tests written in Java
-- [Apache](https://github.com/apache/commons-io) - The Apache Commons IO library contains utility classes, stream implementations, file filters, file comparators, endian transformation classes, and much more.
```
Comment (naivekook) [inline README.md]: we have a lot of other apache libraries
```
implementation("org.apache.commons:commons-lang3:3.12.0")
implementation("org.apache.commons:commons-text:1.9")
implementation("org.apache.commons:commons-csv:1.9.0")
implementation("commons-validator:commons-validator:1.7")
```

File: README.md [hunk 2]
```diff
@@ -105,21 +105,15 @@ If you spot any libraries we added or we don't use anymore, feel free to update
 
 - [Dagger 2](https://github.com/google/dagger) - A fast dependency injector for Android and Java
 - [Retrofit](https://square.github.io/retrofit/) - Retrofit turns your REST API into a Java interface
-- [OkHttp](https://github.com/square/okhttp) - An HTTP+SPDY client for Android and Java applications
```
Comment (naivekook) [inline README.md]: we don't use it directly, but we use it within the retrofit

File: README.md [hunk 3]
```diff
@@ -105,21 +105,15 @@ If you spot any libraries we added or we don't use anymore, feel free to update
 
 - [Dagger 2](https://github.com/google/dagger) - A fast dependency injector for Android and Java
 - [Retrofit](https://square.github.io/retrofit/) - Retrofit turns your REST API into a Java interface
```
Comment (teolemon) [inline README.md]:
```suggestion
- [Retrofit](https://square.github.io/retrofit/) - Retrofit turns your REST API into a Java interface
- [OkHttp](https://github.com/square/okhttp) - An HTTP+SPDY client for Android and Java applications
```

File: README.md [hunk 4]
```diff
@@ -105,21 +105,15 @@ If you spot any libraries we added or we don't use anymore, feel free to update
...
-- [OkHttp](https://github.com/square/okhttp) - An HTTP+SPDY client for Android and Java applications
 - [Mockito](https://github.com/mockito/mockito) - Most popular Mocking framework for unit tests written in Java
```
Comment (teolemon) [inline README.md]:
```suggestion
- [Mockito](https://github.com/mockito/mockito) - Most popular Mocking framework for unit tests written in Java
- [Apache](https://github.com/apache/commons-io) - The Apache Commons IO library contains utility classes, stream implementations, file filters, file comparators, endian transformation classes, and much more.
```

**Comment by teolemon** [comment]
updated @naivekook @VaiTon

**Review by naivekook — APPROVED** [review]

**Pull request merged**